doc/source/api/index.rst

```diff
@@ -43,6 +43,14 @@ system for reads and variants.
 .. toctree::
    references
 
+Sequence Annotations
```

Comment:
The name was chosen because SequenceAnnotations will include more than Features;  The second data type is continuous (wiggle) was not include in this PR to speed implementation.